### ISSUE TITLE
Remove url inputs from loader plugins

### ISSUE THREAD

**Issue by fthr** [vizorvr/vizor#542]
The URL input that these plugins have should be removed for now. Right now that's a very untested feature and probably has CORS issues etc.

That's for all the plugins under the "Load Content" category in plugins.json

Loading content from other web services via API's is something we will be implementing at some point and the tasks for that will be ticketed at  that time.

**Comment by mattatgit** [comment]
![texture](https://cloud.githubusercontent.com/assets/6858525/10135225/53f258ee-662f-11e5-9414-84a972445d93.png)

So was thinking for all loaders there should be nice default previews (so one for texture, one for 3D models, one for video etc), rather than just pink transparency grids. 

This is the one for Texture (i've licensed this)
![vizor_texture_default](https://cloud.githubusercontent.com/assets/6858525/10135274/b48a4702-662f-11e5-8699-9a1ed59080dd.jpg)

**Comment by esaruoho** [comment]
@fthr i on the other hand need URL-inputs to be in "Load Content/Texture" - I have a JSON full of images, ready to be parsed into URLs for each texture loader - and this'd greatly increase the possibilities of using Vizor Patches to quickly create quick VR galleries of images.

The file seems to be in vizor2/vizor/browser/plugins/url_texture_generator.plugin.js but the 

this.input_slots is completely empty. I tried adding 

>                 this.input_slots = [{name: 'url',
>                 dt: core.datatypes.TEXT,
>                 desc: 'Absolute URL to navigate to, eg. http://vizor.io/',
>                def: null}]

but since I'm not able to follow instructions so I could build a local version of this, I can't immediately test and see if URL-to-texture-loading works 100%.

**Comment by esaruoho** [comment]
Hi @mattatgit 
Your
![texture](https://cloud.githubusercontent.com/assets/6858525/10135225/53f258ee-662f-11e5-9414-84a972445d93.png)

Looks like
![vizor](https://user-images.githubusercontent.com/4966687/27740244-01124aba-5dba-11e7-8c08-0842b6940273.png)
on my local build of Vizor Patches

**Comment by mattatgit** [comment]
@esaruoho Weeeeel it doesn't really look great like this. Image should go to edge of box, ie under the port name.

**Comment by esaruoho** [comment]
@mattatgit i think what's going on is that I'm running a version of Texture which has "URL input" enabled - as opposed to the regular version running on Patches, which provides edge-to-edge texture displaying..

<img width="218" alt="7xk___vizor" src="https://user-images.githubusercontent.com/4966687/27759297-18b8bc86-5e36-11e7-97bf-7d3fc8977336.png">

**Comment by mattatgit** [comment]
yep, but the way the system should handle that is to put the image to the edge & overlay the port name, adding a slight text shadow on the port name perhaps to keep it readable.

**Comment by esaruoho** [comment]
@mattatgit I hear ya! Will try and see if I can reach some sort of person who knows a great deal about CSS - it's probably more a CSS issue. Anyway, the Texture-object will require a bit of massaging to get it going, but will try and get all of these things baked in :)

**Comment by esaruoho** [comment]
@mattatgit just to clarify, when you wrote "should be nice default previews, rather than just pink transparency grids" - did you mean that "pink transparency grid" == "no_texture.png" (the pink checkerboard) should be replaced with your aurora-borealis image, so make matt's licensed aurora-borealis image as the new no_texture.png ?

**Comment by mattatgit** [comment]
@esaruoho I think we've discussed this elsewhere, but no, the aurora image should just be for the default image inside image loader patches, not in the editor canvas.
